encrypt_util.go

```diff
@@ -74,21 +74,28 @@ func encryptStream(
 	chunk := make([]byte, chunkSize)
 
 	// encrypt file with CBC
+	padded := false
```

Comment:
Shouldn't this be within the `for` loop? You want to make sure that the last chunk you read was padded or not, but once you set the padding to true it would remain true for the entirety of the for loop.

Reply:
The padding needs to be added to the end of the file if it's never padded.
If it has been padded once, no extra padding is required. This logic is taken from the Python driver.
https://github.com/snowflakedb/snowflake-connector-python/blob/6afaa3025874c8e925caa0cbd6a57f2b62049707/src/snowflake/connector/encryption_util.py#L102

Reply:
Got it